sandbox/public/index.html

```diff
@@ -10,8 +10,8 @@
           content="default-src 'self';
             script-src 'self' 'unsafe-inline';
             style-src 'self' 'unsafe-inline';
-            img-src 'self' http://edgegateway.azurewebsites.net/dests https://edgegateway.azurewebsites.net/dests;
-            connect-src 'self' https://edgegateway.azurewebsites.net http://ex-edge.stable-stage.aam-npe.adobeinternal.net http://localhost:8080"/>
+            img-src *;
```

Comment:
The property configured in the konductor has destinations that load images from a bunch of different URLs. For this reason, I've changed img-src to a wildcard.